sensor/common/heritage/mock.go

```diff
@@ -0,0 +1,15 @@
+package heritage
+
+import "context"
+
+type MockData struct {
+	Data []*PastSensor
+}
+
+func (h *MockData) GetData(_ context.Context) []*PastSensor {
+	return h.Data
+}
+func (h *MockData) HasCurrentSensorData() bool {
+	return true
```

Comment:
Why not `false`?

Reply:
To not change the behavior of existing tests that do not rely on this part of the code. Do you want me to explain that in a comment?